vendor/build-logger/src/ldlogger-tool-gcc.c

```diff
@@ -168,6 +169,9 @@ static void getDefaultArguments(const char* prog_, LoggerVector* args_)
   int incStarted = 0;
 
   strcpy(command, prog_);
+  /* WARNING: this always gets the C++ compiler include
```

Comment:
Shouldn't we just skip logging this?

Reply:
Sorry my previous comment was a bit misleading. Shouldn't we just remove the include path extraction code from the logger?

Reply:
We may need it later. I would leave it here as an option. we will document this.